contrib/babelfishpg_common/sql/geography.sql

```diff
@@ -443,16 +442,63 @@ CREATE OR REPLACE FUNCTION sys.ST_zmflag(sys.GEOGRAPHY)
 	AS '$libdir/postgis-3', 'LWGEOM_zmflag'
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
 
-CREATE FUNCTION sys.ST_Equals(leftarg sys.GEOGRAPHY, rightarg sys.GEOGRAPHY)
+CREATE OR REPLACE FUNCTION sys.STArea(sys.GEOGRAPHY)
+	RETURNS float8
+	AS '$libdir/postgis-3','ST_Area'
+	LANGUAGE 'c' IMMUTABLE STRICT;
+
+CREATE OR REPLACE FUNCTION sys.STSrid(sys.GEOGRAPHY)
+	RETURNS integer
+	AS '$libdir/postgis-3','LWGEOM_get_srid'
+	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.STEquals(geom1 sys.GEOGRAPHY, geom2 sys.GEOGRAPHY)
+	RETURNS sys.BIT
+    AS $$
+    DECLARE
+		srid1 integer;
+		srid2 integer;
+		Equals_result integer;
```

Comment:
No need to declare unused variable.

Reply:
+1, and I think we need none of them since we can directly do:
```
IF STSrid(geom1) != STSrid(geom2) THEN
```

Reply:
Resolved